### PR TITLE
Add text-to-speech beta.

### PR DESCRIPTION
**Pull request by lukesneeringer**

### CODE REVIEW

File: texttospeech/setup.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Google LLC
```
Comment (jonparrott) [inline texttospeech/setup.py]: This needs to use our new setup.py format.

File: texttospeech/nox.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    session.run('py.test', '--quiet', os.path.join('tests', 'unit'))
+
+
+@nox.session
+def lint_setup_py(session):
+    """Verify that setup.py is valid (including RST check)."""
+    session.interpreter = 'python3.6'
+    session.install('docutils', 'pygments')
+    session.run('python', 'setup.py', 'check', '--restructuredtext',
+                '--strict')
```
Comment (chemelnucfin) [inline texttospeech/nox.py]: What about the system/cover tests?

File: texttospeech/setup.py (hunk 1 above)
Comment (lukesneeringer) [reply texttospeech/setup.py]: Done. Could you file me a bug to update that in toolkit?

File: texttospeech/nox.py (hunk 2 above)
Comment (lukesneeringer) [reply texttospeech/nox.py]: I am not as worried about coverage for auto-generated APIs. That is something I want to improve, but it is not my immediate priority.

**Pull request merged**